buildSrc/subprojects/performance/performance.gradle.kts

```diff
@@ -3,8 +3,15 @@ apply(plugin = "org.gradle.kotlin.kotlin-dsl")
 dependencies {
     api(project(":integrationTesting"))
     implementation(project(":build"))
+    implementation("org.openmbee.junit:junit-xml-parser:1.0.0")
     implementation("org.codehaus.groovy.modules.http-builder:http-builder:0.7.2") {
         // Xerces on the runtime classpath is breaking some of our doc tasks
         exclude(group = "xerces")
     }
 }
+
+tasks.withType<Test>().configureEach {
+    if (JavaVersion.current().isJava9Compatible) {
+        jvmArgs("--add-modules", "java.xml.bind")
```

Comment:
What happens if we don't add the module?

Reply:
Some `ClassNotFoundException` because the xml parser is using Java xml API.

Reply:
Will that cause problems when running the performance test task on Java 9?

Reply:
I understand that. We are not passing the `--add-modules` jvm argument to the distributed test task that runs in the main build, do we? That will cause problems when we switch to Java 9, right?

Reply:
Good catch. The behaviour depends on which version the coordinator test runs on. I guess currently we're just trying to migrate all builds to Java 9 but keep all tests on the same Java version as before. But still thanks, I need to take care in another PR.

Reply:
Note that the Test task will run on the Gradle Daemon JVM (soon Java 9) while the actual tests will run on the test JVM which probably is still Java 8. So we will need to add the `--add-modules` as a daemon flag.

Reply:
Every task's actions run in the daemon Jvm, the test task is no difference. The test task forks some JVMs to run the actual tests. Or in the case of the distributed performance test, it starts some TC builds.

Reply:
Oh I see, thanks!

Reply:
Since [it already depends on JAXB api](https://mvnrepository.com/artifact/org.openmbee.junit/junit-xml-parser/1.0.0), the only thing we need to do for Java 9 is to add the missing `javax.activation:activation` API. Also, `lombok` need to be upgraded to support Java 9. Now we no longer need `--add-modules`.